### DISCUSSION TITLE
Implicit integration in Peridigm

### DISCUSSION THREAD

**Discussion by guangdongzhang** [peridigm/peridigm#257] [Q&A]
Hello,
      Are there any Implicit integration examples in Peridigm? It seems most cases are explicit.

**Comment by ppandit95** [comment]
Hi @guangdongzhang 

Can you elaborate more on what do you mean by implicit integration as if you are looking for implicit time integration then yes as on now explicit methods are only available.

Hope this helps.

Regards
Pushkar

**Reply by ZYuan123** [reply]
Hi @ppandit95 ,

it nice to have you in Peridigm community.

since you have answered majority of discussions, i believe I may get some constructive comment from you. Would you please have a look at my question https://github.com/peridigm/peridigm/discussions/299.

thanks in advance ,
Yuan
